openscenario/openscenario_interpreter/src/syntax/performance.cpp

```diff
@@ -21,12 +21,13 @@ inline namespace syntax
 {
 Performance::Performance(
   const Double max_acceleration, const Double max_acceleration_rate, const Double max_deceleration,
-  const Double max_deceleration_rate, const Double max_speed)
+  const Double max_deceleration_rate, const Double max_speed, const Double max_jerk)
 : max_acceleration(max_acceleration),
   max_acceleration_rate(max_acceleration_rate),
   max_deceleration(max_deceleration),
   max_deceleration_rate(max_deceleration_rate),
-  max_speed(max_speed)
+  max_speed(max_speed),
+  max_jerk(max_jerk)
```

Comment:
maxJerk field does not exist in [Performance.](https://www.asam.net/static_downloads/ASAM_OpenSCENARIO_V1.2.0_Model_Documentation/modelDocumentation/content/Performance.html)
Please remove it.